api/HTMLScriptElement.json

```diff
@@ -417,6 +417,53 @@
           }
         }
       },
+      "innerText": {
```

Comment:
So for this one (and textContent), the properties were around since forever, but inherited. Since this update only applies to the HTMLScriptElement I have added it here as though it were a new property.

Is this OK? I mean it looks like you can only access this from current version, which might be confusing to readers.

Reply:
I think this will be confusing, yes, because `innerText` is also defined and BCD-ified in the parent `HTMLElement`: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement#browser_compatibility.

And the `innerText` page takes its BCD from there: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/innerText#browser_compatibility so this change won't even show up in that page.

Maybe instead of this, add a subfeature to HTMLElement.innerText, like: "Script elements can accept a `TrustedScript`"? Then it will show up as a subfeature in https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/innerText#browser_compatibility .

Reply:
It depends on how important strict following of IDL is for BCD. I like the idea though - changed in https://github.com/mdn/browser-compat-data/pull/25839/commits/df3f74ffb58877febef5afef1cbbe0a6c892b35b